README.md

```diff
@@ -1 +1,65 @@
-# collectd-couchbase
\ No newline at end of file
+# Collectd Couchbase Plugin
+
+## Introduction
+
+collectd-couchbase is a [collectd](http://www.collectd.org/) plugin that
+collects statistics from Couchbase.
+
+
+## Requirements
+
+* Collectd 4.9 or later (for the Python plugin)
+* Python 2.7 or later
+* Couchbase 4.1 or later
+
+## Configuration
+
+The following mandatory configuration options describe how the plugin will
+connect to the Couchbase Server:
+
+* Username - the username for authentication to selected bucket, default is None. 
+If your bucket has not set up username and password just ignore this parameter
+* Password - the password for authentication to selected bucket, default is None
+If your bucket has not set up username and password just ignore this parameter
+* Host - Required option. Hostname or IP address of the Couchbase server, default is 'localhost'
+* Port - Required option. The port of the Couchbase server, default is '8091'
+* Interval - interval between sync metrics calls, default is 10 seconds
+
+The following boolean configuration options may be added to enable collection
+of specific statistics:
+
+* CollectTarget - Required option. define the plugin running instance, has two options: 'NODE' - 
```

Comment:
If this is required, it should go in the "mandatory" section just above. If others are optional, be sure to say what value it will have by default if not specified.